includes/storage/SQLStore/SMW_SQLStore3_Queries.php

```diff
@@ -858,53 +858,45 @@ protected function compileValueDescription(
 
 		// Try comparison based on value field and comparator,
 		// but only if no join with SMW IDs table is needed.
-		if ( $diType != SMWDataItem::TYPE_WIKIPAGE ) {
```

Comment:
Where did this go?

https://scrutinizer-ci.com/g/SemanticMediaWiki/SemanticMediaWiki/code-structure/master/class/%2Bglobal%5CSMWSQLStore3QueryEngine